### PR TITLE
Remove UWP references, data binding is for Win32 and UWP Xaml apps

### PR DESCRIPTION
**Pull request by ChrisGuzak**
want to make it clear this applies to all Xaml usage, not just UWP

### CODE REVIEW

File: windows-apps-src/data-binding/data-binding-in-depth.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 ms.assetid: 41E1B4F1-6CAF-4128-A61A-4E400B149011
 title: Data binding in depth
-description: Learn how to use data binding in Universal Windows Platform (UWP) applications
+description: Learn how to use data binding in Xaml applications
```
Comment (alvinashcraft) [inline windows-apps-src/data-binding/data-binding-in-depth.md]: Minor suggestion: XAML is caps in most of the docs.

**Review by alvinashcraft — APPROVED** [review]
Thanks for this. There are a number of pages that need similar updates. Our team has started to analyze which pages in this repo should apply to both UWP and WinUI.

**Comment by Jak-MS** [comment]
IMPORTANT: When this content is ready to merge, the content author must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: windows-apps-src/data-binding/data-binding-in-depth.md (hunk 1 above)
Comment (Jak-MS) [inline windows-apps-src/data-binding/data-binding-in-depth.md]:
```suggestion
description: Learn how to use data binding in XAML applications

```

**Pull request merged**